src/rebuild/srv.c

```diff
@@ -823,10 +823,17 @@ rebuild_scan_broadcast(struct ds_pool *pool,
 	rsi->rsi_tgts_num = tgts_failed->pti_number;
 	rsi->rsi_rebuild_op = rebuild_op;
 	crt_group_rank(pool->sp_group,  &rsi->rsi_master_rank);
+
+retry:
 	rc = dss_rpc_send(rpc);
 	rso = crt_reply_get(rpc);
 	if (rc == 0)
 		rc = rso->rso_status;
+	if (rc == -DER_BUSY) {
+		D_DEBUG(DB_REBUILD, "retry broadcast scan "DF_UUID"\n",
+			DP_UUID(pool->sp_uuid));
+		goto retry;
```

Comment:
Should we dss_sleep for a short while, or that wouldn't make any sense?

Reply:
oh, this will not hog cpu after this rebuild scan rpc is handled, scheduler will reschedule other ult anyway.

Reply:
Compared to hogging the local CPU, I was more concerned about sending at full speed the retry RPCs, most of which end up getting -DER_BUSY.

Reply:
hmm, this stage is supposed to be very short.  but probably adding a ult sleep would not hurt here. I will retry this later. thanks.